app/Makefile

```diff
@@ -61,8 +61,17 @@ ifndef ICONNAME
 $(error ICONNAME is not set)
 endif
 
+# Compile textual mode for all devices excetpt Nano S, 
+# and define a Min stack size for Nano S with some margin
+# to get an error if app grows too much
+ifneq ($(TARGET_NAME),TARGET_NANOS)
+    DEFINES += COMPILE_TEXTUAL
+else
+	APP_STACK_SIZE :=
```

Comment:
APP_STACK_MIN_SIZE can be used for all the devices. The SDK will define the real stack size.
If zxlib was updated to the latest version, I think that we can remove this `APP_STACK_SIZE :=`